content/versionedFeedId.ttl

```diff
@@ -1,12 +1,12 @@
 @prefix : <http://glossary.orcfax.io/#> .
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 :versionedFeedId
-    skos:prefLabel "versioned feed id"@en;
+    skos:prefLabel "versioned feed ID"@en;
     skos:definition '''
-      The extension of the feed id to contain version
+      The extension of the feed ID to contain version
       information.
 
-      It is constructed as `{{feed-id}}/{{feed-version}}`
+      It is constructed as `{{feed-ID}}/{{feed-version}}`
```

Comment:
Strictly speaking there is no declared code convention for this section of text. Its psuedo code embedded in text. So this is not wrong. But I think its very weird to case this as it is now. 
kebab case is generally all lower.

Reply:
fixed